support-frontend/app/controllers/RedemptionController.scala

```diff
@@ -127,21 +142,27 @@ class RedemptionController(
           userHasSub =>
             if (userHasSub)
               Future.successful(redirectToExistingThankYouPage)
-            else
-              tryToShowProcessingPage(redemptionCode)
+            else {
+              identityService.getUser(request.user.minimalUser).value.flatMap{
```

Comment:
I don't think we need this call to identity here, we are doing it in `tryToShowProcessingPage` below

Reply:
ooh good spot there! 👍